keccak256/src/gates/rho.rs

```diff
@@ -240,29 +240,28 @@ mod tests {
             in_state[5 * x + y] = biguint_to_f(&s0_arith[(x, y)]).unwrap();
         }
         let s1_arith = KeccakFArith::rho(&s0_arith);
-        let mut out_state: [pallas::Base; 25] = [pallas::Base::zero(); 25];
+        let mut out_state: [Fp; 25] = [Fp::zero(); 25];
         for (x, y) in (0..5).cartesian_product(0..5) {
             out_state[5 * x + y] = biguint_to_f(&s1_arith[(x, y)]).unwrap();
         }
-        let circuit = MyCircuit::<pallas::Base> {
+        let circuit = MyCircuit::<Fp> {
             in_state,
             out_state,
         };
-        #[cfg(feature = "dev-graph")]
-        {
-            use plotters::prelude::*;
-            let root =
-                BitMapBackend::new("rho-test-circuit.png", (4096, 65536))
-                    .into_drawing_area();
-            root.fill(&WHITE).unwrap();
-            let root = root.titled("Rho", ("sans-serif", 60)).unwrap();
-            halo2::dev::CircuitLayout::default()
-                .render(&circuit, &root)
-                .unwrap();
-        }
+        // #[cfg(feature = "dev-graph")]
+        // {
+        //     use plotters::prelude::*;
+        //     let root =
+        //         BitMapBackend::new("rho-test-circuit.png", (4096, 65536))
+        //             .into_drawing_area();
+        //     root.fill(&WHITE).unwrap();
+        //     let root = root.titled("Rho", ("sans-serif", 60)).unwrap();
+        //     halo2::dev::CircuitLayout::default()
+        //         .render(&circuit, &root)
+        //         .unwrap();
+        // }
```

Comment:
Why has this been commented out?? Not sure I follow TBH. 

I think it has to do with the comment above. We should merge your halo2 work first into our fork and then, use that fork here as dependency. 

Then you'll not need to comment out things like this.

Reply:
It seems the API was changed but I fixed.
I will comment from the next time.